scripts/format.sh

```diff
@@ -8,5 +8,5 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 cd $DIR/../
 
 isort --indent=4 -rc setup.py conftest.py benchmarks examples xain_fl docs/conf.py
-black --exclude "xain_fl/grpc/.*_pb2.*" setup.py conftest.py benchmarks examples xain_fl docs/conf.py
-clang-format -style="{Language: Proto, BasedOnStyle: Google}" -i  protobuf/xain_fl/grpc/*.proto
+black --exclude "xain_fl/cproto/.*_pb2.*" setup.py conftest.py benchmarks examples xain_fl docs/conf.py
+clang-format -style="{Language: Proto, BasedOnStyle: Google}" -i  protobuf/xain_fl/cproto/*.proto
```

Comment:
We can remove clang-format. It wasn't being used properly, and AFAIK, we're moving the proto files to a separate repo, meaning that the linting would have to happen somewhere else

Reply:
Also, please update the CI to reference the new directory:
```suggestion
~/.github/workflows/main.yml
```

Reply:
thanks! removed clang_format and rebased with master to get the new CI dir